scripts/package.json

```diff
@@ -11,6 +11,9 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "fs": "0.0.1-security",
```

Comment:
we will probably need to update the shrinkwrap. You'll need to run npmx generate from the root.